multerConfig/multerConfig.js

```diff
@@ -0,0 +1,24 @@
+import path from "path";
+import multer from "multer";
+import { uuid } from "uuidv4";
+
+const uploadDir = path.join(process.cwd(), "tmp");
+const storeImage = path.join(process.cwd(), "public/avatars");
+
+const storage = multer.diskStorage({
+  destination: (req, file, cb) => {
+    cb(null, uploadDir);
+  },
```

Comment:
```suggestion
  destination: uploadDir,
```

Reply:
Myślę sobie że jeżeli nie używasz cb w jakiejś specyficzny sposób to wystarczy po prostu dać Wskazanie na folder

Reply:
dzięki, poprawione